plugins/ScheduledReports/javascripts/pdf.js

```diff
@@ -122,10 +131,12 @@ function initManagePdf() {
 
         apiParameters.parameters = getReportParametersFunctions[apiParameters.reportType]();
 
+        var hour = adjustHourToTimezone($('#report_hour').val());
```

Comment:
Please provide `-timeZoneDifference` as a second argument for `adjustHourToTimezone()`.